docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -856,6 +904,14 @@ be good (or is it phonetic errors?) A study of exact linkage on some
 large databases relevant to voting in Texas identified address numeric
 data as more accurate than the street name part.
 
+To add noise to the DOB data, I will approximately follow the
+frequencies that Buzz Campbell measured in his BHDS unduplication
+work: Exact match for 96% of DOB, 2 of 3 fields exactly match for 3%,
+day and month fields transposed for 0.2%.  He found 0.24% missing, but
+I will leaving missing for outside of the DOB noise method for now,
+and perhaps make it a configurable option in the future.  Actually, it
+could be sensible to make all of these values configurable options in
+the future.
```

Comment:
So 99.2% will be either exact match, 2/3 match, or day and month transposed. what happens with the remaining .8%?

Reply:
#924 gets these details into viavrium-research.